src/components/ArticleCard.jsx

```diff
@@ -20,25 +23,37 @@ const ArticleCard = ({ article }) => {
         >
           {article.title}
         </Card.Header>
+       
         <Card.Meta
           data-cy="sub-title"
           style={{
             fontWeight: "bold",
             fontSize: 20,
-            color: "black",
-            textAlign: "left",
+            color: "#4f5359",
+            textAlign: "justified",
+            marginBottom: 15,
+            lineHeight: 1.2,
           }}
         >
           {article.sub_title}
         </Card.Meta>
-        <Card.Meta data-cy="author-email">{article.author}</Card.Meta>
-        <Card.Meta data-cy="created-at">{`${t ("CreatedAt")} ${article.created_at}`}</Card.Meta>
-        <Button
+        {/* <Card.Meta data-cy="author-email">{article.author}</Card.Meta> */}
```

Comment:
This one is commented out! Would you want to keep or remove it?

Reply:
O shoots, we missed that one! I have taken it away now, thank you!